Food.css

```diff
@@ -0,0 +1,196 @@
+*
+{
+    padding: 0;
+    margin: 0;
+    text-decoration: none;
+    list-style-type: none;
+    font-family: Georgia, 'Times New Roman', Times, serif;
+}
+:root
+{
+    --white: #ffffff;
+    --lightgrey: #d3d3d3;
+    --darkgrey: #58595d;
+}
+html
+{
+    font-size: 16px;
+}
+body
+{
+    padding: auto;
+}
+header
+{
+    width: 100%;
+    height: 50px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    background-color: var(--white);
+    box-shadow: 0px 0px 0px 2px lightgrey;
+    border: none;
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 999;
+}
+header .logo
+{
+    margin-left: 2em;
+    flex-basis: 0.5;
+    width: 100%;
+    letter-spacing: 0.3em;
+}
+header nav
+{
+    display: flex;
+    flex-basis: 0.5;
+    align-items: center;
+    justify-content: end;
+    width: 100%;
+    height: 100%;
+}
+header nav ul
+{
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    height: 80%;
+}
+header nav ul li
+{
+    display: flex;
+    margin-right: 1em;
+    letter-spacing: 0.2em;
+    cursor: pointer;
+    align-items: center;
+    padding-left: 1em;
+    padding-right: 1em;
+    height: 100%;
+}
+header nav ul li:hover
+{
+    background-color: var(--lightgrey);
+    transition: background-color 0.3s ease;
+}
+section.banner
+{
+    width: 100%;
+    height: 100vh;
+    position: relative;
+}
+section.banner img
+{
+    object-fit: cover;
+    width: 100%;
+    height: 100%;
+}
+section.banner .btitle
+{
+    position: absolute;
```

Comment:
Avoid absolute positioning for responsive design.

Reply:
Okay bhaiya

Reply:
Kindly resolve the issues in the code and commit